style.css

```diff
@@ -46,6 +77,12 @@ section {
   padding: 10px;
 }
 
+.about {
+  display: flex;
+  align-items: center;
+  padding: 30px;
+}
+
```

Comment:
- [ ] Good job implementing the about section, but you still have some issues that need to be fixed. You should make sure your design stick as much as possible to the Figma design 
    - the about section has the border-radius to the right 
    - between the works section and about section there is some space
    - The `Get my resume` button does not have the same color as in the Figma design
Please kindly fix all the points mentioned above 🙃 😉 

Your design

<img width="352" alt="Screen Shot 2022-09-22 at 13 11 58" src="https://user-images.githubusercontent.com/74501523/191735745-743e6ecc-cc42-4443-90da-27664b574a1b.png">

Figma design

<img width="352" alt="Screen Shot 2022-09-22 at 13 12 29" src="https://user-images.githubusercontent.com/74501523/191735802-9098d017-c812-4449-843e-8fad46409878.png">